public/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
```

Comment:
this change should not be reflecting on your PR. Please revert it to its original state